ui/lib/config-ui/addon/components/messages/page/details.hbs

```diff
@@ -32,18 +32,18 @@
   </ToolbarActions>
 </Toolbar>
 
-{{#each @message.allFields as |attr|}}
-  {{#if (or (eq attr.name "endTime") (eq attr.name "startTime"))}}
+{{#each this.orderedKeys as |key|}}
```

Comment:
This felt to me like something that could be managed in components instead since it doesn't have anything to do with the form where the fields are defined. I added an array of keys to loop here which was low effort to refactor.

Reply:
This makes sense, we've done this a lot throughout newer components in the app. I think the pattern is to label this `showFields` or `displayFields` and then `as |field|` since words like "key" and "data" can be really general. But that's definitely a nit.

Reply:
Great that's good to know there will be some familiarity with this already! I updated it to `displayFields` and `field`.